src/spec.js

```diff
@@ -1,30 +1,49 @@
 import test from 'tape';
-import { double, doubleXTimes, doubleEach } from './index';
+import {Animal, Reptile, Primate} from './index';
 
-test( 'double fn', function ( test ) {
-  const actual = double( 5 );
-  const expected = 10;
+test('instance', function (test) {
+    const animal = new Animal();
+    test.ok( animal instanceof Animal, 'should create an instance of Animal' );
 
-  test.equal( actual, expected, 'should double the value' );
-
-  test.end();
+    test.end();
 });
 
-test( 'doubleXTimes', function ( test ) {
-  const actual = doubleXTimes( 5, 3 );
-  const expected = 40;
+test('sound', function (test) {
 
-  test.equal( actual, expected, 'should double 5 three times' );
+    const animal = new Animal();
+    const expected = 'generic sound';
+    const actual = animal.speak('generic sound');
 
-  test.end();
+    test.equal( actual, expected, 'should make a generic sound when it speaks' );
+    test.end();
 });
 
-test( 'doubleEach', function ( test ) {
-  const actual = doubleEach([ 0, 1, 2 ]);
-  const expected = [ 0, 2, 4 ];
+test('reptile', function (test) {
 
-  test.deepEqual( actual, expected, 'should double each in the array' );
+    let actual, expected;
 
-  test.end();
-});
+    const reptile = new Reptile();
+
+    test.ok( reptile instanceof Reptile, 'should be an instance of Reptile' );
+    test.ok( reptile instanceof Animal, 'should be an instance of Animal' );
+
+    expected = Reptile.SOUND;
+    actual = reptile.speak();
+    test.equal( actual, expected, 'should make a reptile sound when it speaks' );
```

Comment:
It is failing because you said `this.SOUND` in your constructor, where `this` refers to the "instance" of the reptile, whereas `Reptile` does not refer to the instance but to the constructor. Does that difference make sense?

As a first hint, if you can access it via `Reptile.SOUND`, how might you set it?